config/config.go

```diff
@@ -42,7 +42,7 @@ func Get() (*Config, error) {
 		BindAddr:                  ":23100",
 		Brokers:                   []string{"localhost:9092"},
 		DatasetAPIURL:             "http://localhost:22000",
-		ElasticSearchAPIURL:       "http://localhost:9200",
+		ElasticSearchAPIURL:       "http://localhost:10200",
```

Comment:
Only question I have is are we explicitly setting this in the app secrets config? If not that then this would break dev/prod. I'm sure we are but best to check.